float8_experimental/float8_linear_utils.py

```diff
@@ -65,6 +72,26 @@ def _update_history_with_new_amax(new_amax, amax_history):
     amax_history.copy_(new_amax_history)
 
 
+def _update_history_stack(
+    new_amax: torch.Tensor, amax_history_stack: torch.Tensor
+) -> torch.Tensor:
+    """
+    Updates `amax_history` (the last N cur_amax values) inplace with the value
+    of `new_amax`.
+
+    Args:
+        new_amax (torch.Tensor): The new amax value to add to the history. (n_amaxes, 1)
+        amax_history_stack (torch.Tensor): The history of amax values. (n_amaxes, history_length)
+    """
+    assert amax_history_stack.dim() == 2, "amax_history_stack must be 2D"
```

Comment:
nit: Include the `amax_history_stack.shape` in the assert error message? (similarly for below assert)